SimCalorimetry/HGCalSimAlgos/test/HGCSiNoiseMapAnalyzer.cc

```diff
@@ -82,11 +84,15 @@ HGCSiNoiseMapAnalyzer::HGCSiNoiseMapAnalyzer(const edm::ParameterSet &iConfig) {
 
   noiseMaps_[DetId::HGCalEE] = std::unique_ptr<HGCalSiNoiseMap>(new HGCalSiNoiseMap);
   noiseMaps_[DetId::HGCalEE]->setDoseMap(doseMapURL, doseMapAlgo);
+  noiseMaps_[DetId::HGCalEE]->setDoseMap(doseMapURL, doseMapAlgo);
```

Comment:
Duplicate line

Reply:
Thanks will address once back to a better keyboard

Reply:
done.